pkg/master/master_test.go

```diff
@@ -526,7 +526,7 @@ func testInstallThirdPartyAPIListVersion(t *testing.T, version string) {
 			setupEtcdList(fakeClient, "/ThirdPartyResourceData/company.com/foos/default", test.items)
 		}
 
-		resp, err := http.Get(server.URL + "/apis/company.com/" + version + "/namespaces/default/foos")
+		resp, err := http.Get(server.URL + "/apis/company.com/group/" + version + "/namespaces/default/foos")
```

Comment:
I also had concerns about groups containing slashes, per discussion in https://github.com/kubernetes/kubernetes/pull/12951#issuecomment-137334362
Can you break down which parts of this url is considered the prefix, and which is considered the group?